### DISCUSSION TITLE
SNES Repro message ID Type 1: 9090, ID Type 2: 9090

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#275] [Need help]
Hi Sanni,

I have this message : 

Flashrom Writer 8bit
ID Type 1: 9090 
ID Type 2: 9090
UNKOWN FLASHROM
![IMG_7756](https://user-images.githubusercontent.com/38383423/136700190-08f7647f-c886-4c36-a68b-021349e2acfd.jpg)

It's my custom PCB with :
- Flash memory **MX29LV640** or **MX29LV800**, 
- bi-directional Transceivers Logic - Translators, Level Shifters (3v<->5v) **LVC4245** and **ALVC164245**, 
- and 100ohm resistors

What reason can it be ?
What can disturb reader to recognize chip?
![stanmoi25-SNES reproduction PCB](https://user-images.githubusercontent.com/38383423/137041030-a23fe790-46ed-4e76-816a-f67c8d76d0c9.jpg)

**Comment by sanni** [comment]
I have seen the 9090 error before but I can't remember where or how it was fixed. Probably best to attach one of those cheap logic analyzers to the 29LV800 and check that WE/CE/OE are triggered correctly. And also check that the voltage at the 29LV is correct.

**Reply by StanMOI** [reply]
Thanks SANNI,
Please let me know if you will find how it was solved.

About **WE** of flash, it's OK, it's tied to **WR** (pin 54 of cartridge edge).
About **CE** of flash, it's should be OK too, its trigged to **/RD** (pin 23 of cartridge edge)
About **OE** of Level Shifters i'am not sure...
Should se Function Table to understand how to use them to get worked in two directions.

**Reply by trgkaigit** [reply]
Please update this thread if you figure this out.  I have a repro giving the same 9090 ID UNKNOWN FLASHROM error with a 29LV160, and confirming a valid connection existed between WE and Pin 54 didn't solve it.

**Reply by StanMOI** [reply]
Can you show Picture of your PCB ?

My error problem is still not resolved.
I’am not sure if « level shifters » have right connections.
It’s my custom PCB, so it can be not perfect cause it’s my first version 🙂
